tests/unit_tests/crud/test_missing_attributes.py

```diff
@@ -0,0 +1,96 @@
+import uuid
+
+from elasticsearch_dsl import AttrDict
+
+from app.api.collections.missing_attributes import (
+    missing_attributes_search,
+    missing_attributes_spec,
+    missingPropertyFilter,
```

Comment:
if this is only used here, then we can as well inline it here! :)

Reply:
It is only used in the API, but for two different cases, see the new endpoint. I am unsure of a good way to inline it there and I would like to keep as much details out of the api.py as possible.
Oh and it is used in the tests as well :D